experimental/sdk/webpubsub-graphql-subscribe/src/WpsWebSocketServer.ts

```diff
@@ -0,0 +1,112 @@
+import { WebPubSubServiceClient } from "@azure/web-pubsub";
+import { WebPubSubEventHandler } from "@azure/web-pubsub-express";
+import { ApolloServer} from "apollo-server-express";
+import { execute, GraphQLSchema, subscribe } from "graphql";
+import { SubscriptionServer } from 'subscriptions-transport-ws';
+import express from 'express';
+import WebSocket from "ws";
+import { WpsPubSub } from "./azure-wps-pubsub";
+import {config} from "./utils"
+
+/**
+  * a virtual WebSocket.Server without real server
+ */
+class VirtualWebSocketServer extends WebSocket.Server{
+	protocol: string = "graphql-ws";
+	readyState: number = WebSocket.OPEN;
+
+	constructor() {
+		 super({port: undefined, noServer: true}); 
+	}
+}
+
+/** 
+  * A VirtualWebSocketServer which is binded to an Azure Web PubSub client connection according to `servecClient` and `connectionId`
+  * It overrides method `send` and send all message to its binded connection in JSON format.
+ */
+class SubWpsWebSocketServer extends VirtualWebSocketServer{
+	serviceClient: WebPubSubServiceClient;
+	connectionId: string;
+
+	constructor(serviceClient: WebPubSubServiceClient, connectionId: string) {
+		super();
+		this.serviceClient = serviceClient;
+		this.connectionId = connectionId;
+	}
+
+	send(data: any) { this.serviceClient.sendToConnection(this.connectionId, data, { contentType: "text/plain" }); }
+}
+
+/** 
+  * A `VirtualWebSocketServer` which replaces original `WebSocket.Server` to communicate between the server and WebPub service using HTTP protocol
+  * `connectionId_to_ws` records the mapping from the `connectionId` of each client to its corrsponding `SubWpsWebSocketServer`.
+  * Finally `SubWpsWebSocketServer` sends messages to Web PubSub service.
+  * 
+ */
+class WpsWebSocketServer extends VirtualWebSocketServer {
+	app: any;
+	connectionId_to_ws: { [key:string]: SubWpsWebSocketServer } = {};
```

Comment:
let's follow the general JS naming convention: camel case parameter name

Reply:
all similar problems fixed